k9mail/src/main/java/com/fsck/k9/activity/ActivityListener.java

```diff
@@ -1,5 +1,6 @@
 package com.fsck.k9.activity;
 
+
```

Comment:
Unnecessary new line

Reply:
It was added by Android Studio's "reformat code" action using our code style scheme. Apparently our settings file includes this:

```xml
    <option name="BLANK_LINES_AFTER_PACKAGE" value="2" />
```